engine/src/flow/raster_cache.cc

```diff
@@ -16,15 +16,28 @@ namespace uiwidgets {
 RasterCacheResult::RasterCacheResult(sk_sp<SkImage> image,
                                      const SkRect& logical_rect)
     : image_(std::move(image)), logical_rect_(logical_rect) {}
+#if __ANDROID__
+#define ABS(x) abs(x)
+template<typename T>
+T abs(T x) {
+  if (x < 0) {
+    return -x;
+  }
+  return x;
+}
+#else 
+#define ABS(x) std::abs(x)
+#endif
 
 void RasterCacheResult::draw(SkCanvas& canvas, const SkPaint* paint) const {
   TRACE_EVENT0("uiwidgets", "RasterCacheResult::draw");
   SkAutoCanvasRestore auto_restore(&canvas, true);
   SkIRect bounds =
       RasterCache::GetDeviceBounds(logical_rect_, canvas.getTotalMatrix());
+      auto a = bounds.size().width() - image_->dimensions().width();
   FML_DCHECK(
-      std::abs(bounds.size().width() - image_->dimensions().width()) <= 1 &&
-      std::abs(bounds.size().height() - image_->dimensions().height()) <= 1);
+        ABS(bounds.size().width() - image_->dimensions().width()) <= 1 &&
```

Comment:
the indent is not good.

btw, maybe we should rename ABS => Math_abs() ? ABS is not a good name here